os/board/common/crashdump.c

```diff
@@ -40,10 +42,12 @@
 #if defined(CONFIG_BOARD_DUMP_UART)
 #define RAMDUMP_HANDSHAKE_STRING	"TIZENRTRMDUMP"
 #define FSDUMP_HANDSHAKE_STRING	"TIZENRTFSDUMP"
+#define EXTFSDUMP_HANDSHAKE_STRING	"TIZENRTEXTFSD"
 #define TARGET_REBOOT_STRING		"TIZENRTREBOOT"
 #define HANDSHAKE_STR_LEN_MAX		(13)
 #define USERFS_PART_NAME		"user"
 #define USERFS_PART_NAME_LEN		(4)
+#define PROCFS_ENTRY_NAME_LEN		(40)
```

Comment:
We already have a definition reagarding length of name in limit.h
CONFIG_NAME_MAX..

Reply:
CONFIG_NAME_MAX often defined as 32 is barely enough for this procfs entry
`/proc/fs/smartfs/smart1p1/dump_all` -> 34 characters (35 including '\0' for snprintf)
`/proc/fs/smartfs/smart0p8/dump_lsector` -> 38 characters

Reply:
No, there's limit value about file name & path name.
So this is unncessary.
please see limit.h one more time.

Reply:
Yes,
CONFIG_NAME_MAX doesn't suffice,
But CONFIG_PATH_MAX will, lets use that